lib/packages_groups_xml_creator.py

```diff
@@ -67,7 +99,39 @@ def create_group_xml(group_name, group_pkgs):
     return grp
 
 
-def create_comps_xml(packages_groups):
+def create_environment_xml(environment_name, environment_groups):
+    """
+    Construct an XML object representing an 'environment' element.
```

Comment:
You could explain explicitly that an environment is simply a named list of packages groups somewhere.